tower/CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 # Unreleased
 
+- **util**: Add `CloneService` which is a `Clone + Send` boxed `Service`.
```

Comment:
Here and elsewhere in the comments you refer to `CloneService`, but I think you mean `CloneBoxService`?
```suggestion
- **util**: Add `CloneBoxService` which is a `Clone + Send` boxed `Service`.
```

Reply:
Ah yes good catch! I was a bit too quick on the auto complete.